app/views/user_mailer/_sharing.html.erb

```diff
@@ -5,9 +5,6 @@
   <a href='<%= twitter_share_url(@actionPage) -%>'>
     <%= image_tag 'share-on-tw-thin.png', alt: 'Share on Twitter' -%>
   </a>
-  <a href='<%= google_share_url(@actionPage) -%>'>
```

Comment:
we can also get rid of this method which is in `app/helpers/action_page_helper.rb`, then good to merge!